deploy/osfoffline-setup.iss

```diff
@@ -25,7 +25,7 @@ DisableProgramGroupPage=true
 DisableStartupPrompt=true
 LicenseFile=C:\Users\IEUser\Documents\GitHub\OSF-Offline\LICENSE
 OutputDir=C:\Users\IEUser\Desktop
-OutputBaseFilename=OSF-Offline-Setup-v0.1.0
+OutputBaseFilename=OSF-Sync-Windows-v0.3.0
```

Comment:
Can we just use the `MyAppVersion` variable in all the places where we are currently manually editing the `0.3.0` string?